### PR TITLE
Adds information to README.md file how to use vue-devtools

### PR DESCRIPTION
**Pull request by StCyr**
Signed-off-by: Cyrille Bollu <cyrpub@bollu.be>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,27 @@ If you want to work on improving the components it’s best to run the latest co
     - In the repository of an app do `npm link @nextcloud/vue` (you need to re-link any time you do `npm ci` in the app)
 4. Then build the app with: `npm run build` (or watch for changes with `npm run watch`)
 
+### Using vue-devtools
+
+If you want to use [vue-devtools](https://github.com/vuejs/vue-devtools) in your browser, you need to patch your nextcloud instance as follow:
```
Comment (skjnldsv) [inline README.md:56]: This is only for firefox

Comment (StCyr) [reply README.md:56]: Oh yes, they said something about that in the chat.

Is there something else to do for Chrome,... or nothing needed for those browsers?

Comment (skjnldsv) [reply README.md:56]: no, works for chrome :)

Comment (StCyr) [reply README.md]: ok, PR updated

File: README.md [hunk 2]
```diff
@@ -51,6 +51,27 @@ If you want to work on improving the components it’s best to run the latest co
...
+	 class ContentSecurityPolicy extends EmptyContentSecurityPolicy {
+	        /** @var bool Whether inline JS snippets are allowed */
+	-       protected $inlineScriptAllowed = false;
+	+       protected $inlineScriptAllowed = true;
+	        /** @var bool Whether eval in JS scripts is allowed */
+	-       protected $evalScriptAllowed = false;
+	+       protected $evalScriptAllowed = true;
+	        /** @var array Domains from which scripts can get loaded */
+	        protected $allowedScriptDomains = [
+	                '\'self\'',
```
Comment (skjnldsv) [inline README.md:73]: Maybe add some code block with diff linting? Dunno if github supports it right away :thinking:

Comment (StCyr) [reply README.md:73]: I don't understand sorry :(

Comment (skjnldsv) [reply README.md:73]: a block quote https://stackoverflow.com/a/40883538/3885878

Comment (StCyr) [reply README.md]: yes it works!

I've updated my PR

**Review by skjnldsv — APPROVED** [review]

**Review by GretaD — APPROVED** [review]

**Pull request merged**